lib/client-api/src/client_api.test.ts

```diff
@@ -652,12 +652,7 @@ describe('preview.client_api', () => {
           expect(entry.stories).toHaveLength(1);
           expect(entry.stories[0].name).toBe('story');
 
-          // v3 returns the same function we passed in
```

Comment:
I'm not sure why these tests tested both v3+v4 of Storybook (how is that possible?)